pkg/translator/jaeger/jaegerproto_to_traces.go

```diff
@@ -274,16 +274,17 @@ func setInternalSpanStatus(attrs pcommon.Map, dest ptrace.SpanStatus) {
 	statusExists := false
 
 	if errorVal, ok := attrs.Get(tracetranslator.TagError); ok {
+		statusCode = ptrace.StatusCodeOk
 		if errorVal.BoolVal() {
```

Comment:
What if the attribute value is not of the boolean type? In this solution, we always take it as `error=false` which might be incorrect. I believe we should ignore any non-boolean values here. WDYT?

Reply:
sounds valid. i placed there an additional condition. :+1: